eng/Versions.props

```diff
@@ -23,6 +23,11 @@
     <IsServicingBuild Condition=" '$(PreReleaseVersionLabel)' == 'servicing' ">true</IsServicingBuild>
     <!-- This is used for error checking to ensure generated code and baselines are up to date when we increment the patch. -->
     <PreviousExtensionsReleaseVersion Condition=" '$(PatchVersion)' != '0' ">$(MajorVersion).$(MinorVersion).$([MSBuild]::Subtract($(PatchVersion), 1))</PreviousExtensionsReleaseVersion>
+    <!--
+        When StabilizePackageVersion is set to 'true', this branch will produce stable outputs for 'Shipping' packages
+    -->
+    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">true</StabilizePackageVersion>
+    <DotNetFinalVersionKind Condition="'$(StabilizePackageVersion)' == 'true'">release</DotNetFinalVersionKind>
```

Comment:
This'll work but it seems much safer to make `DotNetFinalVersionKind` a variable in azure-pipelines.yml. Even better if that variable's visible when queueing the pipeline.

Separately is `$(StabilizePackageVersion)` used anywhere else?

Reply:
I feel like this is a better place because it is right alongside the rest of the branding?

And no, StabilizePackageVersion is not used elsewhere.

Reply:
I withdraw my comment